serving/metadata-webhook/config/webhook.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       serviceAccountName: controller
       containers:
       - name: webhook
-        image: registry.ci.openshift.org/openshift/openshift-serverless-nightly:metadata-webhook
+        image: registry.ci.openshift.org/knative/openshift-serverless-nightly:metadata-webhook
```

Comment:
I suppose we moved to knative nad it has the latest. Are there any other images that need change?

Reply:
As far as I checked. https://github.com/openshift/release/pull/35057 is one of them.